README.md

```diff
@@ -64,6 +64,7 @@ az deployment group create -f ./main.json -g my-rg
   * Previously all parameters had to be declared together in one `"parameters": {}` object, variables had to be declared together in one `"variables": {}` object, etc.
 * Automatic dependency management in certain scenarios. Bicep will automatically add `dependsOn` in the compiled ARM Template if the symbolic name is used in another resource declaration.
 * Richer validation and intellisense than what is available in the ARM Tools VS Code extension. For example, in bicep we have intellisense on GET properties (`output sample string = resource.properties.*`)
+* All resource type properties that exist via Azure Resource Manager are available in bicep. Since bicep produces ARM template `.json` files there are no descrepencies of what properties and settings are available. 
```

Comment:
I don't know if this is an argument that life is better with bicep, but is the case that it is equally good as ARM Templates in this sense. Though, it is a reason why life is better than bicep in comparison to other IaC tools. Thoughts?

Added a comma and fixed a spelling error below:

All resource type properties that exist via Azure Resource Manager are available in bicep. Since bicep produces ARM template `.json` files, there are no discrepancies of what properties and settings are available.

Reply:
Yeah...fair challenge.  I wasn't sure either where it makes the most sense; however, after dealing more and more with other IaC solutions like Terraform felt it is an important advantage (or disadvantage depending on who you ask) that's important to realize when evaluating it.  Thought was if someone is evaluating or determining to use bicep they were going to hone in on this section.  That being said I went back and scanned over the ReadMe and notices it is called out in the last line of the What Is Bicep section.